src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -5,32 +5,28 @@
 /**
  * All the constants to be uses for the robot
  */
-public abstract class RobotConstants extends RobotMap {
-    public LimelightConstants limelightConstants = new LimelightConstants();
-
-    public TesterConstants testerConstants = new TesterConstants();
-    public VisionConstants visionConstants = new VisionConstants();
```

Comment:
i'm pretty sure even if the class is static you still need to instantiate the classes

Reply:
nope